front/pages/api/v1/w/[wId]/spaces/[spaceId]/data_sources/[dsId]/tables/[tId]/index.ts

```diff
@@ -198,13 +198,15 @@ async function handler(
 
       return res.status(200).json({
         table: {
-          name: table.name,
+          name: table.title ?? table.name,
```

Comment:
not sure to understand the `table.title ??` thing. Why ?

Reply:
The idea is to deprecate name and have people switch to title instead, that's why I give this order of priority

Reply:
Ah I wasn't aware of that, that's good to know!
I still lack context but I don't see why users should provide a db name if they already provide a title, can't we simply come up with a name ourself based (or not) on the title ?

Reply:
users I don't know, but for connectors it may be different.
Eg a notion database might have a display title and a "short slugified, SQL-safe name".

Not sure in practice how this is being done currently, maybe I am wrong and this is actually always the same thing